### PR TITLE
Change tutorial dataset to "moving pictures"

### PR DESCRIPTION
**Pull request by cameronmartino**
### Update tutorial to moving pictures for a two reasons:
1. Smaller file sizes
2. Flows better with existing QIIME tutorials 

other updates:

- changed links in readme to relect this
- fixed broken link to QIIME plugin

### CODE REVIEW

File: ipynb/tutorials/moving-pictures.md [hunk 1]
```diff
@@ -34,76 +34,78 @@ To demonstrate this in action we will run an example dataset below, where the ou
 
 ## Example 
 
-In this example we will Robust Aitchison PCA via deicode on a study of sleep apnea published in [Tripathi et al. (2018)](https://msystems.asm.org/content/3/3/e00020-18). The study consists of mouse fecal samples and focuses on comparing the gut microbiome of animals exposed to intermittent hypoxia and hypercapnia (IHH; as a model of obstructive sleep apnea) to controls exposed to room air (air). 
+In this example we will Robust Aitchison PCA via DEICODE on the “Moving Pictures” tutorial, if you have not yet completed the tutorial it can be found [here](https://docs.qiime2.org/2019.1/tutorials/moving-pictures/). The dataset consists of human microbiome samples from two individuals at four body sites at five timepoints, the first of which immediately followed antibiotic usage ([Caporaso et al. 2011](https://www.ncbi.nlm.nih.gov/pubmed/21624126)). If you have completed this tutorial run the following command and skip the download section.
```
Comment (ElDeveloper) [inline ipynb/tutorials/moving-pictures.md]: we will Robust -> we will use Robust

File: ipynb/tutorials/moving-pictures.md [hunk 2]
```diff
@@ -34,76 +34,78 @@ To demonstrate this in action we will run an example dataset below, where the ou
...
 
-![](https://cdn1.imggmi.com/uploads/2019/2/6/eaf9c58ee3b00949fcd4947333376a03-full.png)
+![](http://i66.tinypic.com/28rm5pj.png)
 
-From this visualization we noticed that exposure_type seems explain the clusters well. We can run [PERMANOVA](https://docs.qiime2.org/2019.1/plugins/available/diversity/beta-group-significance/) on the distances to get a statistical significance for this. 
+From this visualization we noticed that BodySite seems explain the clusters well. We can run [PERMANOVA](https://docs.qiime2.org/2019.1/plugins/available/diversity/beta-group-significance/) on the distances to get a statistical significance for this.
```
Comment (ElDeveloper) [inline ipynb/tutorials/moving-pictures.md]: seems explain -> seems to explain

**Review by ElDeveloper — APPROVED** [review]
Looks good, a couple of suggested changes but otherwise this looks good.

**Comment by cameronmartino** [comment]
@ElDeveloper  Typos fixed, should be good to go, thanks!

**Pull request merged**